containers/ome-tiff-offsets/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# We just need to use --file to point at it, instead of assuming it is in context.
```

Comment:
I don't think this is true any longer? the Dockerfile is in context, like normal, now, I think?

Reply:
I just copied this over, I wasn't sure what it meant.  Could you clarify?  Should we keep it?